front/item_disk.form.php

```diff
@@ -39,16 +39,19 @@
 if (!isset($_GET["id"])) {
    $_GET["id"] = "";
 }
-if (!isset($_GET["computers_id"])) {
-   $_GET["computers_id"] = "";
+if (!isset($_GET["items_id"])) {
+   $_GET["items_id"] = "";
+}
+if (!isset($_GET["itemtype"])) {
+   $_GET['itemtype'] = '';
 }
 
-$disk = new ComputerDisk();
+$disk = new Item_Disk();
 if (isset($_POST["add"])) {
    $disk->check(-1, CREATE, $_POST);
 
    if ($newID = $disk->add($_POST)) {
-      Event::log($_POST['computers_id'], "computers", 4, "inventory",
+      Event::log($_POST['items_id'], $_POST['itemtype'], 4, "inventory",
                  //TRANS: %s is the user login
                  sprintf(__('%s adds a volume'), $_SESSION["glpiname"]));
```

Comment:
actually not in the pr purpose, but a ``getLinkURL`` instead of ``getFormURL`` is the attended behavior